tests/test_retry_plugin.py

```diff
@@ -940,8 +945,8 @@ def test_eventually_passes():
     assert len([line for line in result.outlines if line.startswith('\t  File')]) == len(verbosity)
 
 
-@mark.skipif(xdist_installed is False, reason="Only run if xdist is installed locally")
-def test_xdist_reporting_compatability(testdir):
+@xdist_test_marker
+def test_xdist_reporting_compatibility(testdir):
```

Comment:
Ha, thank you. I think I've fixed this typo twice on different branches only to not merge it